src/Apps/Components/AppToasts.tsx

```diff
@@ -9,7 +9,14 @@ export const AppToasts: React.FC = () => {
   if (toasts.length === 0) return null
 
   return (
-    <Box position="fixed" zIndex={Z.toasts} bottom={0} left={0} width="100%">
+    <Box
+      position="fixed"
+      zIndex={Z.toasts}
+      bottom={0}
+      left={0}
+      width="100%"
+      style={{ pointerEvents: "none" }}
```

Comment:
if we add this, then clicking on toast (for example, to hide the toast message) will be ignored. for this reason, this solution is not suitable

Reply:
fixed [here](https://github.com/artsy/force/pull/12515/commits/cd4b389e218d5226900900e81e27c529a3437f46) by using another `pointer-events` for children content